### PR TITLE
Allow configurable Shards & Replicas on Contacts Index

### PR DESCRIPTION
**Pull request by tybritten**

### CODE REVIEW

File: indexers/contacts.go [hunk 1]
```diff
@@ -6,14 +6,15 @@ import (
 	_ "embed"
 	"encoding/json"
 	"fmt"
-	"time"
-
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"time"
```
Comment (rowanseymour) [inline indexers/contacts.go:11]: gofmt should have `time` up above...

File: indexers/contacts.go [hunk 2]
```diff
@@ -47,6 +48,9 @@ func (i *ContactIndexer) Index(db *sql.DB, rebuild, cleanup bool) (string, error
 
 	// doesn't exist or we are rebuilding, create it
 	if physicalIndex == "" || rebuild {
+		_ = json.Unmarshal(contactsSettingsFile, &contactsSettings)
```
Comment (rowanseymour) [inline indexers/contacts.go]: should handle this error

**Review by rowanseymour — CHANGES_REQUESTED** [review]

File: indexers/contacts.go (hunk 1 above)
Comment (tybritten) [reply indexers/contacts.go:11]: it kept moving it to the bottom 🤷🏻

File: indexers/contacts.go (hunk 2 above)
Comment (rowanseymour) [reply indexers/contacts.go]: ah just noticed this is unmarshaling an embedded file so shouldn't ever error - anyways I'll merge and tweak to use `jsonx.MustUnmarshal`

**Pull request merged**